automation-testing/src/main/java/stepDefinitions/ChromeDriverManager.java

```diff
@@ -0,0 +1,18 @@
+package stepDefinitions;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import stepDefinitions.DriverManager;
+
+public class ChromeDriverManager implements DriverManager {
+
+    private WebDriver driver;
+
+    public WebDriver getDriver() {
+
+        System.setProperty("webdriver.chrome.driver","C:\\SeleniumDrivers\\chromedriver.exe");
```

Comment:
Store the drivers in a folder inside the framework structure, and call them from there, so this setting does not need to be set manually.